### PR TITLE
Tc/json response

### PR DESCRIPTION
**Pull request by CedThen**

### CODE REVIEW

**Comment by CedThen** [comment]
3rd

**Comment by CedThen** [comment]
-standardized useMutation responses, although one of them doesn't work properly
-standardized error responses
-fixed time cell text color
-fixed role cell colors in edit mode

File: server/routes/schedules.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ router.get('/schedule', certify, async (req, res, next) => {
   try {
     const { scheduleId } = req.query;
     const response = await retrieveOneSchedule(scheduleId);
-    if (!response) return res.status(404).send({ message: 'No schedules found' });
+    if (!response) return next({ status: 404, message: 'No schedule found' });
```
Comment (showduhtung) [inline server/routes/schedules.js:30]: nice

File: server/routes/schedules.js [hunk 2]
```diff
@@ -38,12 +38,9 @@ router.get('/schedule', certify, async (req, res, next) => {
 router.post('/schedule', certify, async (req, res, next) => {
   try {
     const { churchId, title } = req.body;
-    if (await doesScheduleExist(churchId, title)) {
-      // TODO clean up and make consistent responses
-      res.statusMessage = 'Schedule already exists';
-      res.status(409).send();
-      return;
-    }
+    if (await doesScheduleExist(churchId, title))
```
Comment (showduhtung) [inline server/routes/schedules.js:41]: you can if (await)?

File: server/services/schedules.js [hunk 3]
```diff
@@ -41,6 +41,7 @@ const updateRouter = {
   deletedServices: deleteServices,
   deletedEvents: deleteEvents,
   dataModel: updateModel,
+  scheduleId: () => {},
```
Comment (showduhtung) [inline server/services/schedules.js:44]: this will be a function?

**Review by showduhtung — APPROVED** [review]
lgtm

**Pull request merged**